SPECS/xs-openssl.spec

```
@@ -524,6 +538,34 @@ export LD_LIBRARY_PATH
 %postun libs -p /sbin/ldconfig
 
 %changelog
+* Fri Sep 08 2023 David Morel <david.morel@vates.fr> - 1:1.1.1k-9.1
+- Sync with Centos 8 Stream's 1.1.1k-9.
+- Remove update-expired-SCT-certificates patch as it is integrated upstream
+- *** Upstream changelog ***
+- * Wed Feb 08 2023 Dmitry Belyavskiy <dbelyavs@redhat.com> - 1:1.1.1k-9
+- - Fixed Timing Oracle in RSA Decryption
+    Resolves: CVE-2022-4304
+- - Fixed Double free after calling PEM_read_bio_ex
+    Resolves: CVE-2022-4450
+- - Fixed Use-after-free following BIO_new_NDEF
+    Resolves: CVE-2023-0215
+- - Fixed X.400 address type confusion in X.509 GeneralName
+    Resolves: CVE-2023-0286
+
+- * Thu Jul 21 2022 Dmitry Belyavskiy <dbelyavs@redhat.com> - 1:1.1.1k-8
+- - Fix no-ec build
+    Resolves: rhbz#2071020
+
+- * Tue Jul 05 2022 Clemens Lang <cllang@redhat.com> - 1:1.1.1k-7
+- - Fix CVE-2022-2097: AES OCB fails to encrypt some bytes on 32-bit x86
+    Resolves: CVE-2022-2097
+- - Update expired certificates used in the testsuite
+    Resolves: rhbz#2092462
+- - Fix CVE-2022-1292: openssl: c_rehash script allows command injection
+    Resolves: rhbz#2090372
+- - Fix CVE-2022-2068: the c_rehash script allows command injection
+    Resolves: rhbz#2098279
```

If I were to re-establish the changelog embedding syntax, I'd probably do it differently, without all these dashes, but for now the convention is

```suggestion
- *** Upstream changelog ***
- * Wed Feb 08 2023 Dmitry Belyavskiy <dbelyavs@redhat.com> - 1:1.1.1k-9
- - Fixed Timing Oracle in RSA Decryption
-   Resolves: CVE-2022-4304
- - Fixed Double free after calling PEM_read_bio_ex
-   Resolves: CVE-2022-4450
- - Fixed Use-after-free following BIO_new_NDEF
-   Resolves: CVE-2023-0215
- - Fixed X.400 address type confusion in X.509 GeneralName
-   Resolves: CVE-2023-0286
- * Thu Jul 21 2022 Dmitry Belyavskiy <dbelyavs@redhat.com> - 1:1.1.1k-8
- - Fix no-ec build
-   Resolves: rhbz#2071020
- * Tue Jul 05 2022 Clemens Lang <cllang@redhat.com> - 1:1.1.1k-7
- - Fix CVE-2022-2097: AES OCB fails to encrypt some bytes on 32-bit x86
-   Resolves: CVE-2022-2097
- - Update expired certificates used in the testsuite
-   Resolves: rhbz#2092462
- - Fix CVE-2022-1292: openssl: c_rehash script allows command injection
-   Resolves: rhbz#2090372
- - Fix CVE-2022-2068: the c_rehash script allows command injection
-   Resolves: rhbz#2098279
```

---

Thanks, I was unsure about it, fixed.